src/tweak.h

```diff
@@ -117,23 +123,36 @@ class CTweakRef : public CTweakBase
             tweaks.erase(CTweakKey(name));
     }
 
-    CTweakRef(const char *namep, const char *helpp, DataType *val, EventFn callback = NULL)
+    CTweakRef(const char *namep, const char *helpp, DataType *val, EventFn callback = nullptr)
         : name(namep), help(helpp), value(val), eventCb(callback)
     {
         tweaks[CTweakKey(name)] = this;
     }
 
-    CTweakRef(const std::string &namep, const std::string &helpp, DataType *val, EventFn callback = NULL)
+    CTweakRef(const std::string &namep, const std::string &helpp, DataType *val, EventFn callback = nullptr)
         : name(namep), help(helpp), value(val), eventCb(callback)
     {
         tweaks[CTweakKey(name)] = this;
     }
 
-    virtual std::string GetName() const { return name; }
-    virtual std::string GetHelp() const { return help; }
-    virtual UniValue Get() const { return UniValue(*value); }
+    virtual std::string GetName() const
+    {
+        std::unique_lock<std::mutex> lck(cs_tweak);
+        return name;
+    }
+    virtual std::string GetHelp() const
```

Comment:
What's the reason to lock getting the `help` and `name` of the tweak? Isn't that ialways expected to be immutable data that can accessed in parallel?

(Not that this would likely be any performance issue either way, just wondering)